cybergis_compute_client/CyberGISCompute.py

```diff
@@ -67,7 +68,63 @@ def __init__(self, url="cgjobsup.cigi.illinois.edu", port=443, protocol='HTTPS',
         self.job = None
         self.recentDownloadPath = None
 
-    def login(self, manualLogin=True, verbose=True):
+    def encrypt_token(self, token):
```

Comment:
Be sure to document all of these functions with docstrings and check that they render nicely.